opennms-doc/guide-admin/src/asciidoc/text/flows/setup.adoc

```diff
@@ -216,3 +216,25 @@ admin@opennms()> config:edit org.opennms.features.flows.persistence.kafka
 admin@opennms()> config:property-set bootstrap.servers 127.0.0.1:9092
 admin@opennms()> config:update
 ----
+
+=== Correcting clock skew
+
+Flow analyses uses timestamps exposed by the underlying flow management protocol.
+These timestamps will be set depending on the clock of the exporting router.
+If the routers clock differs from the actual time, this will be reflected in received flows and therefore skew up further analysis and aggregation.
```

Comment:
> If the router's clock differs from the actual time, this will be reflected in received flows and therefore skew up further analysis and aggregation.